client/lib/wp/sync-handler/whitelist-handler.js

```diff
@@ -0,0 +1,31 @@
+/**
+ * Module dependencies
+ */
+
+import debugFactory from 'debug';
+const debug = debugFactory( 'calypso:sync-handler:whitelist' );
+
+const whitelist = [
+	'^/batch',
+	'^/me',
+	'^/products$',
+	'^/read',
+	'^/sites/.+/'
```

Comment:
Are all of these definitely safe? Can we start even more conservatively, with post-list requests for single-site and all-sites? `'^/me/posts', '^sites/[\w.]+/posts'`.